lib/chef/provider/package/apt.rb

```diff
@@ -89,11 +99,19 @@ def locked_packages
         end
 
         def install_package(name, version)
-          package_name = name.zip(version).map do |n, v|
-            package_data[n][:virtual] ? n : "#{n}=#{v}"
+          if source_files_exist?
+            sources = name.map { |n| name_sources[n] }
+            logger.info("#{new_resource} installing package(s): #{name.join(" ")}")
```

Comment:
I think this is cruft, as the multipackage providers have a log line up in the install action in the parent Package class here: https://github.com/chef/chef/blob/main/lib/chef/provider/package.rb#L122